### PR TITLE
Sort Questions by Whether They are Answered by an Expert

### LINKED ISSUE #18: As a user, I want to be able to filter the questions based on whether they are answered by experts, so that I can find the most reliable answer to my question

### PR DESCRIPTION
**Pull request by RealSnowKid**

### CODE REVIEW

**Review by RobbeBryssinck — APPROVED** [review]

File: Abracadabra/AbracadabraAPI/AbracadabraAPI/Migrations/20201123084159_inital.cs [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 namespace AbracadabraAPI.Migrations
 {
-    public partial class Initial : Migration
+    public partial class inital : Migration
```
Comment (VoidBGS) [inline Abracadabra/AbracadabraAPI/AbracadabraAPI/Migrations/20201123084159_inital.cs:6]: Typo, should be initial

File: Abracadabra/abracadabra_web_app/pages/subject/[subject]/expert.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+import Navbar from 'src/components/Navbar.js';
+import FilterButtons from 'src/components/FilterButtons';
+import QuestionBody from 'src/components/QuestionBody.js';
+import DefaultErrorPage from 'next/error';
+import SubjectService from 'src/services/SubjectService';
+import QuestionService from 'src/services/QuestionService';
+
+function Subject({ subjectName, response }) {
+
+    // if (response === 404 || response == "failure" || response === 400) {
```
Comment (VoidBGS) [inline Abracadabra/abracadabra_web_app/pages/subject/[subject]/expert.js]: Unused code

File: Abracadabra/AbracadabraAPI/AbraCadabraAPITests/Utilities.cs [hunk 3]
```diff
@@ -38,7 +38,7 @@ public static List<ApplicationUser> GetUsers()
         {
             return new List<ApplicationUser>()
             {
-             new ApplicationUser(){Id = "1", UserName = "Ricardo", Email="Ricardo@gmail.com", PasswordHash="123123"},
+             new ApplicationUser(){Id = "1", UserName = "Ricardo", Email="Ricardo@gmail.com", PasswordHash="123123",},
```
Comment (LarsvdBrandt) [inline Abracadabra/AbracadabraAPI/AbraCadabraAPITests/Utilities.cs:41]: Does this comma need to be here?

**Review by LarsvdBrandt — APPROVED** [review]

File: Abracadabra/abracadabra_web_app/pages/subject/[subject]/expert.js (hunk 2 above)
Comment (RealSnowKid) [reply Abracadabra/abracadabra_web_app/pages/subject/[subject]/expert.js]: [d015465](https://github.com/philips-labs/fontys-2020-team-abra-cadabra/pull/56/commits/d01546512aebd06be9d055c912a70d9eb4f2d436)

**Pull request merged**